src/Microsoft.AspNetCore.Antiforgery/Internal/IClaimUidExtractor.cs

```diff
@@ -1,20 +1,21 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
 using System.Security.Claims;
 
 namespace Microsoft.AspNetCore.Antiforgery.Internal
 {
     /// <summary>
-    /// This interface can extract unique identifers for a claims-based identity.
+    /// This interface can extract unique identifers for a list of claims-based identities.
     /// </summary>
     public interface IClaimUidExtractor
     {
         /// <summary>
         /// Extracts claims identifier.
         /// </summary>
-        /// <param name="identity">The <see cref="ClaimsIdentity"/>.</param>
+        /// <param name="identities">The list of <see cref="ClaimsIdentity"/>.</param>
         /// <returns>The claims identifier.</returns>
-        string ExtractClaimUid(ClaimsIdentity identity);
+        string ExtractClaimUid(IEnumerable<ClaimsIdentity> identities);
```

Comment:
Why not just have this take a `ClaimsPrincipal`? It's a bit unclear what the semantics are supposed to be for "list of identities".